product_docs/docs/transporter/ba_preview/getting_started/installing/index.mdx

```diff
@@ -6,32 +6,28 @@ navigation:
   - linux_x86_64
 ---
 
-To install EDB Transporter for internal purposes:
+Select a link to access the applicable installation instructions.
 
-### Accessing CloudSmith
+## Linux [x86-64 (amd64)](linux_x86_64)
 
-Log in to Okta, and navigate to the CloudSmith tile to access the dev CloudSmith repository. 
+### Red Hat Enterprise Linux (RHEL) and derivatives
 
-### Finding the appropriate package
+- [RHEL 8](linux_x86_64/transporter_rhel_8), [RHEL 7](linux_x86_64/transporter_rhel_7)
 
-Look for the last `cdcreader` version for the architecture and operating system you are using.
+- [Oracle Linux (OL) 8](linux_x86_64/transporter_rhel_8), [Oracle Linux (OL) 7](linux_x86_64/ransporter_rhel_7)
 
-### Installing the package
+- [Rocky Linux 8](linux_x86_64/transporter_other_linux_8)
 
-Navigate to the setup page of the selected `cdcreader` and follow the installation instructions.
+- [AlmaLinux 8](linux_x86_64/transporter_other_linux_8)
 
-### Verifying the installation
+- [CentOS 7](linux_x86_64/transporter_centos_7)
 
-To verify the files have been added to your machine, run: 
+### SUSE Linux Enterprise (SLES)
 
-```shell
-sudo find / -name "cdcreader"
-```
+- [SLES 15](linux_x86_64/transporter_sles_15), [SLES 12](linux_x86_64/transporter_sles_12)
 
-The output should look similar to this: 
+### Debian and derivatives
 
-```shell
-/opt/cdcreader
-/opt/cdcreader/usr/bin/cdcreader
-/usr/bin/cdcreader
-```
+- [Ubuntu 22.04](linux_x86_64/transporter_ubuntu_22), [Ubuntu 20.04](linux_x86_64/transporter_ubuntu_20), [Ubuntu 18.04](linux_x86_64/transporter_ubuntu_18)
+
+- [Debian 11](linux_x86_64/transporter_debian_11), [Debian 10](linux_x86_64/transporter_debian_10)
```

Comment:
In the EPAS docs, we also have Debian 12 documented. Just pointing this out here :).

Reply:
This comment comes a bit late, but it looks like Debian 10 is also EOL.

Reply:
Removed Debian11 and added support for Debian12.